prow/plugins/respond.go

```diff
@@ -23,8 +23,13 @@ import (
 	"k8s.io/test-infra/prow/github"
 )
 
+// AboutThisBotWithoutCommands contains the message that explains how to interact with the bot.
 const AboutThisBotWithoutCommands = "Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository."
+
+// AboutThisBotCommands contains the message that links to the command the bot understand.
```

Comment:
commands

If you search it doesn't look like we need to export this constant

Reply:
I remember I found some other exported identifiers that are not used outside their packages. But I rather handle these in another PR.